src/hooks/postMessage.ts

```diff
@@ -314,14 +315,14 @@ const configurePostMessageHooks = (queryConfig: QueryClientConfig) => {
 
     useEffect(() => {
       if (!queryConfig.isStandalone) {
-        const sendHeight = (height: number): void => {
+        const sendHeight = debounce((height: number): void => {
           console.debug('[app-postMessage] Sending height', height);
           console.debug('communication channel is', communicationChannel);
           communicationChannel?.postMessage({
             type: POST_MESSAGE_KEYS.POST_AUTO_RESIZE,
             payload: height,
           });
-        };
+        }, queryConfig.debounceTimeAutoResize);
```

Comment:
I'm wondering if it is necessary to have the `debounceTimeAutoResize` in the `queryConfig`? From my point of view, this parameter has no place in the request client configuration. What do you think?

```suggestion
        }, DEBOUNCE_TIME_AUTORESIZE);
```

Reply:
If we want to use a different debounce per app, we may want to set it directly in the hook like `useAutoResize(id, { autoResizeDebounceTimeMs: 150 }`);